libr/asm/arch/mips/gnu/mips-dis.c

```diff
@@ -430,6 +430,16 @@ const struct mips_arch_choice mips_arch_choices[] =
     mips_cp0sel_names_sb1, ARRAY_SIZE (mips_cp0sel_names_sb1),
     mips_hwr_names_numeric },
 
+  { "loongson2e", 1, bfd_mach_mips_loongson_2e, CPU_LOONGSON_2E,
```

Comment:
this is not used by asm.cpu, right? i mean, the pr only adds the definitions for those mips archs. so it's good to go, but its unused code iiuic